rust/src/transformations/cast_metric/ffi.rs

```diff
@@ -32,17 +32,20 @@ pub extern "C" fn opendp_transformations__make_ordered_random(
     where
         MI::Distance: 'static,
         <MI::OrderedMetric as Metric>::Distance: 'static,
+        (AnyDomain, MI): MetricSpace,
+        (AnyDomain, MI::OrderedMetric): MetricSpace,
     {
         let trans = try_!(super::make_ordered_random::<AnyDomain, MI>(domain));
 
-        FfiResult::Ok(util::into_raw(Transformation::new(
+        Transformation::new(
```

Comment:
In hindsight, it's funny to see how much cleaner stuff like this becomes with the new flavor of `Transformation::new`.

Reply:
It probably should have been `Ok(Transformation::new(...)).into()` before!